src/services/SubmissionService.js

```diff
@@ -247,11 +248,30 @@ async function listSubmissions (authUser, query) {
   const data = await helper.fetchFromES(query, helper.camelize(table))
   logger.info(`listSubmissions: returning ${data.rows.length} submissions for query: ${JSON.stringify(query)}`)
 
-  data.rows = _.map(data.rows, (submission) => {
+  data.rows = await Promise.all(_.map(data.rows, async (submission) => {
+    // Check to see if we've loaded in the submission reviews from OR (PS-268)
+    let hasReviewInES = false
+    if (submission.review) {
+      _.forEach(submission.review, (review) => {
+        if (review.metadata && review.metadata.source) {
+          if (review.metadata.source === 'Online Review') {
+            hasReviewInES = true
+          }
+        }
+      })
+    }
+
+    // We don't have the Online Review details in ES, so go to Informix to get them and
```

Comment:
Really excellent comment for anyone coming to this months or years from now -- nice work.